Readme redistribution and reorganisation

r3nor opened the pull request:
I feel like the current Readme is a bit chaotic in some aspects. I tried to redistribute and make it feel more natural visually and easier to read. I hope you like the changes :)

The information in the Readme is exactly the same!

r3nor committed:
redistribute and organize readme

bo0tzz commented:
While I'll leave it up to Alex to merge this, I think this is a very welcome bit of tidying up!

zackpollard commented on README.md:
```diff
@@ -173,57 +175,31 @@ Run `docker-compose up` or `docker compose up` (based on your docker's version)
-<p align="left">
-  <img src="design/admin-interface.png" width="500" title="Admin User Management">
-<p/>
+| You can add and manage users from de administration page. | <img src="design/admin-interface.png" width="500" title="Admin User Management"> |
```
Typo: "de" should be "the"

zackpollard commented on README.md:
```diff
@@ -14,75 +20,66 @@
+### Mobile
+| | | | | |
+| - | - | - | - | - |
+| <img src="design/login-screen.png" width="150" title="Login With Custom URL"> <p align="center"> Login with custom URL </p> | <img src="design/backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup Settings </p> | <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup tunning </p> | <img src="design/home-sc ...
```
Typo: "Backup tunning" should be "Backup tuning" however I think maybe "Backup Selection" is better?

zackpollard reviewed (changes requested):
Looks much cleaner like this, the contents is also a great addition, thanks for the PR. Will also leave this down to Alex to merge in. A couple of typos in there that i've commented on 😄

r3nor committed:
Fix typos

r3nor commented:
@zackpollard thank you for the corrections, just fixed them!

alextran1502 commented on README.md:
```diff
@@ -14,75 +20,66 @@
+| 🗃️ Support RAW (HEIC, HEIF, DNG, Apple ProRaw) | Yes | Yes
+| 🧭 Metadata view (EXIF, map) | Yes | Yes
+| 🔎 Search by metadata, objects and image tags | Yes | No
+| ⚙️ Administrative functions (user management) | N/A | Yes
```
I love the emoji here, but it will be hard to find the applicable one for new features

r3nor replied:
So you think it is better to remove them?

alextran1502 commented on README.md:
```diff
@@ -14,75 +20,66 @@
+- [Screenshots](#screenshots)
+- [Installation](#installation)
+- [Mobile App](#-mobile-app)
+- [Developers](#development)
```
This would be `Development`?

r3nor replied:
Yes, just fixed it!

alextran1502 commented on README.md:
```diff
@@ -134,10 +136,10 @@ wget -O .env https://raw.githubusercontent.com/immich-app/immich/main/docker/.en
 * Populate `UPLOAD_LOCATION` as prefered location for storing backup assets.
-* Populate a secret value for `JWT_SECRET`
-* [Optional] Populate Mapbox value.
+* Populate a secret value for `JWT_SECRET`, you can use this command: `openssl rand -base64 128`
```
Great addition, thank you

alextran1502 commented on README.md:
```diff
@@ -173,57 +175,31 @@ Run `docker-compose up` or `docker compose up` (based on your docker's version)
+  <br/>  
 
-#### Get the app on Apple AppStore [here](https://apps.apple.com/us/app/immich/id1613945652):
+# 🤳 Mobile app
```
Let's keep the header consistent with other headers that don't have the emoji

alextran1502 commented on README.md:
```diff
@@ -14,75 +20,66 @@
+| | | | | |
+| - | - | - | - | - |
+| <img src="design/login-screen.png" width="150" title="Login With Custom URL"> <p align="center"> Login with custom URL </p> | <img src="design/backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup Settings </p> | <img src="design/selective-backup-screen.png" width="150" title="Backup Setting Info"> <p align="center"> Backup selection </p> | <img src="design/home- ...
+| <img src="design/shared-albums.png" width="150" title="Shared Albums"> <p align="center"> Shared albums </p>  |  <img src="design/nsc6.png" width="150" title="EXIF Info"> <p align="center"> EXIF info </p>  | <img src="https://media.giphy.com/media/y8ZeaAigGmNvlSoKhU/giphy.gif" width="150" title="Loading ~4000 images/videos"> <p align="center"> Loading ~4000 images/videos </p>  |
```
![image](https://user-images.githubusercontent.com/27055614/185256355-716f2b96-72b8-4273-86cb-316a81898833.png)
I think the table format here makes the layout kinda hard to follow as the indentation offset for each cell

alextran1502 replied:
let's try a different ideas of display these messages or we can keep it linearly

r3nor replied:
I just changed it to a linear display with centered images. Maybe you think this is better.

Also removed the `.env` example text and replaced it with a link (that opens in a new tab) to the `.env.example` file that is present in the repo. This way you don't need to change the `.env` in two places and users are linked directly to the file itself. Also this optimizes the space on the readme so steps look more simple at a glance.

alextran1502 commented:
Thank you for a great PR, I added some suggestion, let me know if you have any question

r3nor committed:
Change developers to development

r3nor committed:
Remove mobile app emoji

r3nor committed:
Multiple changes

- Removed the .env example inbody to link to the .env.example file; this way it won't be needed to change the readme if the .env file changes and the readme gets shorter.
- Redistributed the steps:
  - Inline instead of tables
  - bullets in steps
- Fix wording

alextran1502 commented:
Thanks, it looks good now
